### PR TITLE
docs: update Kitsu's CORS field and documentation link

### PR DESCRIPTION
**Pull request by wopian**
Unknown -> Yes

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

**Comment by wopian** [comment]
@yannbertrand any update on this being merged? I'm one of the maintainers of the Kitsu API

**Comment by yannbertrand** [comment]
There is a certificate issue using this URL:

```shell
curl https://docs.kitsu.apiary.io/
```

> curl: (60) SSL: no alternative certificate subject name matches target host name 'docs.kitsu.apiary.io'
> More details here: https://curl.haxx.se/docs/sslcerts.html
>
> curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ API | Description | Auth | HTTPS | CORS |
 | [AniList](https://github.com/AniList/ApiV2-GraphQL-Docs) | Anime discovery & tracking | `OAuth` | Yes | Unknown |
 | [AnimeNewsNetwork](https://www.animenewsnetwork.com/encyclopedia/api.php) | Anime industry news | No | Yes | Yes |
 | [Jikan](https://jikan.moe) | Unofficial MyAnimeList API | No | Yes | Yes |
-| [Kitsu](http://docs.kitsu.apiary.io/) | Anime discovery platform | `OAuth` | Yes | Unknown |
+| [Kitsu](https://docs.kitsu.apiary.io/) | Anime discovery platform | `OAuth` | Yes | Yes |
```
Comment (wopian) [inline README.md:82]:
```suggestion
| [Kitsu](https://kitsu.docs.apiary.io/) | Anime discovery platform | `OAuth` | Yes | Yes |
```

**Comment by wopian** [comment]
My bad, missed the subdomains hadn't been switched around yet here for the docs (Apiary changed that a few years ago)

Docs are https://kitsu.docs.apiary.io/ and APIs are https://kitsu.io/api/edge and https://kitsu.io/api/graphql

**Comment by gaciux** [comment]
Unsubscribe

2020-10-22, kt 19:51, James Harris <notifications@github.com> rašė:

**Comment by atomartem** [comment]
Unsubscribe

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by yannbertrand** [comment]
Thanks

**Pull request closed without merging**